vllm_spyre/platform.py

```diff
@@ -71,8 +71,13 @@ class SpyrePlatform(Platform):
     def device_type(cls):
         # TODO: temporary hack while BertModels
         # inherit SupportsV0Only in vllm upstream.
+        import vllm.model_executor.models as me_models
         from vllm.config import ModelConfig
-        ModelConfig.is_v1_compatible = is_v1_compatible
+
+        # no need to patch after the model_config change
+        if 'model_config' not in \
+                inspect.getfullargspec(me_models.ModelRegistry.is_v1_compatible).args:
+            ModelConfig.is_v1_compatible = is_v1_compatible
```

Comment:
Ah interesting I had this solved in another way - https://github.com/vllm-project/vllm-spyre/pull/341/files#diff-16ac04c4e75668ccde20cd2cfb82fa496d5d98dbe169ffede15652dc60a16066R54-R59